src/lib/caches/WFInfo.js

```
@@ -0,0 +1,89 @@
+
+  static get prices() {
+    if (settings.wfInfo?.prices) {
+      return this.#cache.getKey('prices');
+    }
+    return undefined;
+  }
+
+  static populate(logger = Logger('WFINFO')) {
+    return this.#hydrate(logger);
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)